templates/campaign_results.html

```diff
@@ -109,5 +113,5 @@ <h2>Details</h2>
 <div id="flashes" class="row"></div>
 </div>
 {{end}} {{define "scripts"}}
-<script src="/js/dist/app/campaign_results.min.js"></script>
-{{end}}
\ No newline at end of file
+<script src="/js/src/app/campaign_results.js"></script>
```

Comment:
We'll want to make sure to keep these pointing to the `dist` files, since those are the compiled sources. It's ok if you didn't compile the JS (I do that anyways), but if we could keep the references the same I'd appreciate it 😄

Reply:
Yep, I noticed this one after I made the PR, I've reverted it now. Just makes it easier to do quick changes to the code (especially as gulp stopped working on my server, so I had to minify manually in the end). Anyways, all sorted in the three templates.